modules/statics/src/networks.ts

```diff
@@ -633,6 +633,18 @@ class SeiTestnet extends Testnet implements AccountNetwork {
   explorerUrl = 'https://sei.explorers.guru/transaction/';
 }
 
+class INJ extends Mainnet implements AccountNetwork {
+  name = 'Injective';
+  family = CoinFamily.INJ;
+  explorerUrl = 'https://explorer.injective.network/transaction/';
+}
+
+class InjTestnet extends Testnet implements AccountNetwork {
+  name = 'Testnet Injective';
```

Comment:
```suggestion
  name = 'InjectiveTestnet'; 
```
I think this should be class name as well